homework-2/src/scss/core/_common.scss

```diff
@@ -0,0 +1,61 @@
+*,
+*:after,
+*:before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+a,
+button {
+  background-color: transparent;
+  border: none;
+  outline: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+button{
+  box-shadow: 0px 6px 12px rgba(127, 127, 127, 0.3);
```

Comment:
this is a specific style not for all buttons, so it should be general style, create class for this styles